### DISCUSSION TITLE
How to disable url bar hiding

### DISCUSSION THREAD

**Discussion by jekv2** [bromite/bromite#1158] [Q&A]
Is there a way to stop url bar from hiding when scrolling on a web page? Screen space nowadays is a lot so no need for it, its more work to make it show after is hides.

**Comment by csagan5** [comment]
@uazo perhaps this commit is related? https://github.com/chromium/chromium/commit/0b6d7d68b24de027878692c24048dc14dcc4a435

**Reply by uazo** [reply]
yes, just tried it, that's it

**Reply by jekv2** [reply]
How do i permanently show the url bar?

**Reply by csagan5** [reply]
@uazo what did you try, specifically? Perhaps this could be made as a flag?

@jekv2 you cannot

**Reply by uazo** [reply]
I forced `enableHidingTopControls` to true, the result was that the bar was not hiding.

> Perhaps this could be made as a flag?

Sure, we can.
